jobs/uaa/spec

```diff
@@ -438,6 +438,16 @@ properties:
     description: "This value can be set to the origin key of an identity provider. If set, the user will be directed to this identity provider automatically if no other identity provider is discovered or selected via login_hint. When not set, legacy chained authentication (where uaa is attempted first followed by ldap) is used."
     example: uaa
     default: null
+  login.aliasEntitiesEnabled:
+    description: |
+      Enable the creation, update and deletion of aliases for identity providers and users. The alias feature allows
```

Comment:
Enable the creation, update and deletion for identity providers and users, which have alias configured.

Reply:
fixed with commit 684312bc2342c6a00ebb8b6f9e41c025273b6d6c